docsets/Vulkan/docset.json

```diff
@@ -1,10 +1,20 @@
 {
     "name": "Vulkan",
-    "version": "1.0.38",
+    "version": "1.0.39",
     "archive": "Vulkan.tgz",
```

Comment:
That's the archive for the `specific_version`. You need to have it at `docsets/Vulkan/Vulkan.tgz` too for the main docset.